packages/components/src/components/AssetLogo/AssetLogo.tsx

```diff
@@ -71,10 +71,6 @@ export const AssetLogo = ({
         setIsPlaceholder(true);
     };
 
-    useEffect(() => {
-        setIsPlaceholder(!shouldTryToFetch);
-    }, [shouldTryToFetch]);
```

Comment:
Is it correct? It's not going to update placeholder when you programatically update `shouldTryToFetch `

Reply:
reverted